### PR TITLE
Add all IMPC related attributes to OBA that are not already there

### LINKED ISSUE #188: Add all IMPC related attributes to OBA that are not already there
Add all IMPC related attributes to OBA that are not already in there:

[impc_terms_merged](https://docs.google.com/spreadsheets/d/13UZVY5YxIZom-sS-TDuPSXet8sdtzo1OwQ4LYxrZVrI/edit#gid=0)

In order to get this done column D (pattern), and everything else from B-H should be used.
The goal is to append any traits in here to the pattern data files located in `../patterns/data/default/{pattern}.tsv`

### PR DESCRIPTION
**Pull request by souzadevinicius**
fixes #188

### CODE REVIEW

File: src/patterns/data/default/entity_attribute.tsv [hunk 1]
```diff
@@ -1,4102 +1,4390 @@
 defined_class	defined_class_name	entity	entity_name	attribute	attribute_name	exact_synonyms	xrefs
-OBA:0000036	body fluid levels	UBERON:0006314	bodily fluid	PATO:0000918	volume
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:2]: please make commit in a way that no rows are removed, there should be only additions.

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/patterns/data/default/entity_attribute.tsv [hunk 2]
```diff
@@ -4100,3 +4100,291 @@ OBA:2050052	occipital lobe volume 	UBERON:0002021	occipital lobe	PATO:0000918	vo
 OBA:2050053	lentiform nucleus volume	UBERON:0002263	lentiform nucleus	PATO:0000918	volume
 OBA:2050097	serum amyloid A protein measurement	CHEBI:138152	serum amyloid A	PATO:0000070	amount
 OBA:2050098	serum paraoxonase/arylesterase 1 measurement	PR:000013021	serum paraoxonase/arylesterase 1	PATO:0000070	amount
+OBA:2060000	aggression towards inanimate objects	NBO:0000112	aggressive behavior towards inanimate objects	PATO:0000161	rate
+OBA:2060001	enhanced contextual conditioning behavior	NBO:0000223	contextual conditioning behavior	PATO:0000161	rate
+OBA:2060002	enhanced coordination	NBO:0000339	motor coordination	PATO:0000161	rate
+OBA:2060003	excessive tearing	GO:0070075	tear secretion	PATO:0000161	rate
+OBA:2060004	hyperactivity	NBO:0000338	kinesthetic behavior	PATO:0000161	rate
+OBA:2060005	impaired cued conditioning behavior	NBO:0000224	cued conditioning behavior	PATO:0000161	rate
+OBA:2060006	increased aggression	NBO:0000015	aggressive behavior	PATO:0000161	rate
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:4109]: remove all values from the `defined_class_label` column. They are wrong, and DOS-DP will generate them automatically. Look here, it says "increased aggression". While the trait is about "rate of aggressive behaviour".

File: src/patterns/data/default/entity_attribute.tsv [hunk 3]
```diff
@@ -4100,3 +4100,291 @@ OBA:2050052	occipital lobe volume 	UBERON:0002021	occipital lobe	PATO:0000918	vo
...
+OBA:2060048	abnormal coat/ hair morphology	UBERON:0001037	strand of hair	PATO:0000051	morphology
+OBA:2060049	abnormal hippocampal fimbria morphology	UBERON:0002310	hippocampus fimbria	PATO:0000051	morphology
+OBA:2060050	abnormal optic disk morphology	UBERON:0001783	optic disc	PATO:0000051	morphology
+OBA:2060051	abnormal vitreous body morphology	UBERON:0001798	vitreous body	PATO:0000051	morphology
+OBA:2060052	hydrops fetalis	UBERON:0007221	neonatal stage	PATO:0000025	composition
+OBA:2060053	increased body weight	UBERON:0000468	multicellular organism	PATO:0000125	mass
+OBA:2060054	increased brain weight	UBERON:0000955	brain	PATO:0000125	mass
+OBA:2060055	increased embryo weight	UBERON:0000922	embryo	PATO:0000125	mass
+OBA:2060056	increased epididymis weight	UBERON:0001301	epididymis	PATO:0000125	mass
+OBA:2060057	increased heart weight	UBERON:0000948	heart	PATO:0000125	mass
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:4150]: The deduplication did not work:

```
OBA:VT0007028	heart mass	UBERON:0000948	heart	PATO:0000125	mass
```

Only deduplify on the `entity` and `attribute` columns!

**Review by matentzn — CHANGES_REQUESTED** [review]
Looking better, almost there.

File: src/patterns/data/default/entity_attribute.tsv (hunk 2 above)
Comment (matentzn) [reply src/patterns/data/default/entity_attribute.tsv:4109]: (leave empty value)

**Review by matentzn — APPROVED** [review]
Dont merge without an @rays22 review

File: src/patterns/data/default/entity_attribute_location.tsv [hunk 4]
```diff
@@ -245,3 +245,8 @@ OBA:2050096	serum creatinine amount	CHEBI:16737	creatinine	PATO:0000070	amount	U
 OBA:2020003	chemical in blood amount	CHEBI:24431	chemical entity	PATO:0000070	amount	UBERON:0000178	blood
 OBA:2020004	chemical in serum amount	CHEBI:24431	chemical entity	PATO:0000070	amount	UBERON:0001977	serum
 OBA:2020005	lysine in blood amount	CHEBI:25094	lysine	PATO:0000070	amount	UBERON:0000178	blood
+OBA:2060000		CHEBI:145810	insulin	PATO:0000070	amount	UBERON:0000178	blood
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute_location.tsv:248]: ID overlap with the other spreadsheet

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by ar-ibrahim — APPROVED** [review]

**Review by matentzn — APPROVED** [review]

File: src/patterns/data/default/entity_attribute.tsv [hunk 5]
```diff
@@ -4100,3 +4100,177 @@ OBA:2050052	occipital lobe volume 	UBERON:0002021	occipital lobe	PATO:0000918	vo
 OBA:2050053	lentiform nucleus volume	UBERON:0002263	lentiform nucleus	PATO:0000918	volume
 OBA:2050097	serum amyloid A protein measurement	CHEBI:138152	serum amyloid A	PATO:0000070	amount
 OBA:2050098	serum paraoxonase/arylesterase 1 measurement	PR:000013021	serum paraoxonase/arylesterase 1	PATO:0000070	amount
+OBA:2060000		NBO:0000112	aggressive behavior towards inanimate objects	PATO:0000161	rate
+OBA:2060001		NBO:0000223	contextual conditioning behavior	PATO:0000161	rate
+OBA:2060002		NBO:0000339	motor coordination	PATO:0000161	rate
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:4105]: I wonder about this one @rays22.. 

`motor coordination` is a trait IMO. `rate` of `motor coordination` feels a bit off.. Its not really a behaviour, is an ability.

Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv:4105]: I think it is a valid interpretation of 'motor coordination' to mean a process as defined in NBO.
To me, a quality of 'motor coordination' looks more like a trait.

**Review by rays22 — APPROVED** [review]

**Pull request merged**